papers/core_bench.md

```diff
@@ -0,0 +1,98 @@
+---
+author: Martiño Ríos-García
+date: 2024-11-13
+title-block-banner: core_bench_papers/ai_agents_blog_pic.jpg 
+bibliography: ../references.bib
+---
+
+# CORE-Bench: Fostering the Credibility of Published Research Through a Computational Reproducibility Agent Benchmark
+
+## Why discussing this paper?
+
+I chose Siegel et al.'s paper [@siegel2024corebenchfosteringcredibilitypublished] for our journal club because:
+
+- I really like what this folks are proposing about agents' evaluation.
+- I think that the evaluation of agents is the current bottleneck in the development of new agents.
+- They write some really interesting statements such as *"Before agents can automate scientific research, they must be able to reproduce existing results."*.
+- One of the conclusions is really interesting for one of the projects that I am working on.
```

Comment:
```suggestion
I chose Siegel et al.'s paper [@siegel2024corebenchfosteringcredibilitypublished] for our journal club because:

- I really like what these folks are proposing about agents' evaluation.
- I think that the evaluation of agents is the current bottleneck in the development of new agents.
- They write some really interesting statements such as *"Before agents can automate scientific research, they must be able to reproduce existing results."*.
- One of the conclusions is really interesting for one of the projects that I am working on.
```